project/CelebornBuild.scala

```diff
@@ -202,7 +202,7 @@ object CelebornCommon {
         "io.dropwizard.metrics" % "metrics-core" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-graphite" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-jvm" % metricsVersion,
-        "io.netty" % "netty-all" % nettyVersion,
+        ("io.netty" % "netty-all" % nettyVersion).exclude("io.netty", "netty-handler-ssl-ocsp"),
```

Comment:
I intend to consolidate all dependencies into a global object called `Dependencies`, similar to Maven's `dependencyManagement`, to improve dependency management. cc @pan3793